docs/source/release_notes/release-process.rst

```diff
@@ -5,24 +5,20 @@ Release Process
 Release Notes
 ================
 
-When you make a PR to Atlas, a release notes change must be made to record your change or an explicit reason given why it is unnecessary. At the same time, you should consider whether your change is breaking or not and update the version number appropriately based on the previous release number.
+When you make a PR to Atlas, a release notes change must be made to record your change or an explicit reason given why it is unnecessary.
+Unless the `no changelog` tag is present, the `changelog-bot` will leave an interactive comment on your PR. Please select the appropriate change type as listed in the comment.
 
 Release Schedule
 ================
 
 We aim to release once a week, but may release faster to patch bugs or slower if there is low code churn.
```

Comment:
This is no longer true. More realistic would be "We release as often as we can, so that the change set between releases is small."